src/private_broadcast.h

```diff
@@ -37,7 +38,17 @@ class PrivateBroadcast
     bool Add(const CTransactionRef& tx) EXCLUSIVE_LOCKS_REQUIRED(!m_mutex);
 
     /**
-     * Pick the transaction that has been sent fewest times and least recently.
+     * Forget a transaction.
+     * @param[in] tx Transaction to forget.
+     * @return The number of times the transaction was sent if the transaction existed
+     * and was removed, otherwise empty optional (the transaction was not in the storage).
+     */
+    std::optional<size_t> Remove(const CTransactionRef& tx)
+        EXCLUSIVE_LOCKS_REQUIRED(!m_mutex);
+
+    /**
+     * Pick the transaction with the fewest send attempts, and confirmations,
+     * and oldest send/confirm times.
```

Comment:
to minimize the diff, this should e adjusted in a previous commit where it was introduced (not in 129d67ebafdd63c4a48fc6c8f45c21f729cd8cf3)